imgui/backends/imgui_impl_wgpu.cpp

```diff
@@ -84,6 +84,7 @@ static unsigned int     g_frameIndex = UINT_MAX;
 struct Uniforms
 {
     float MVP[4][4];
+    char _align[256-sizeof(MVP)];
```

Comment:
Rather than fixing the uniform struct, you can simply align the buffer creation and write buffer operations. But it is weird that you need to align to something as high as 256.

Reply:
Indeed, the high number surprised me as well. Wonder what the underlying implementation/hardware is doing.
I came up with this alternative, it seems less brittle to me than keeping track of offsets in GPU-land:
https://github.com/mheistermann/LearnWebGPU-Code/commits/step100-fixes-2
(A separate commit does some refactoring, not sure if for better or worse)